### PR TITLE
Add missing age and sex to participants.tsv file

### LINKED ISSUE #10: missing age and sex for some subjects in participants.tsv
Not all subjects in [participants.tsv](https://github.com/spine-generic/data-multi-subject/blob/master/participants.tsv) file have listed age. Age could be used as a covariate in statistical analysis to rule out potential effect of age on results etc.
Most of subjects with missing age also miss sex.

Sites with missing age:

**institution_id | institution**
- [x] amu | AMU - CEMEREM
- [x] beijing750 | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] beijingPrisma | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] beijingVerio | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] brnoUhb | The University Hospital Brno, Department of Radiology and Nuclear Medicine, Czech Republic
- [x] dresden | Institut_fur_Neuroradiologie, Dresden, Germany
- [x] fslAchieva | Santa Lucia Foundation IRCCS, Rome, Italy
- [x] fslPrisma | Santa Lucia Foundation IRCCS, Rome, Italy - sex and age added, date of scan still missing
- [x] vallHebron | Vall d'Hebron Research Center

### PR DESCRIPTION
**Pull request by valosekj**
This PR is intended for adding of missing age, sex (and date of scan) information to `participants.tsv` file.

Fixes: https://github.com/spine-generic/data-multi-subject/issues/10

### CODE REVIEW

**Comment by valosekj** [comment]
this PR is incorrectly open in `valosekj/data-multi-subject`, it should be opened in `spine-generic/data-multi-subject`. I will open PR correctly when https://github.com/spine-generic/data-multi-subject/pull/11 will be merged into `spine-generic/data-multi-subject` master.

File: participants.tsv [hunk 1]
```diff
@@ -258,4 +258,4 @@ sub-vuiisIngenia02	F	36	2019-02-20	vuiisIngenia	Vanderbilt University Institute
 sub-vuiisIngenia03	F	25	2019-02-20	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
 sub-vuiisIngenia04	F	30	2019-02-20	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
 sub-vuiisIngenia05	M	29	2019-02-25	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
-sub-vuiisIngenia06	M	31	2019-02-26	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
\ No newline at end of file
+sub-vuiisIngenia06	M	31	2019-02-26	vuiisIngenia	Vanderbilt University Institute of Imaging Science	Philips	Ingenia	16-ch neurovascular	Release 5.5.0.0	"K.P. O'Grady, A.J.E. Combes, S.A. Smith"
```
Comment (jcohenadad) [inline participants.tsv:261]: be careful with the added line

Comment (valosekj) [reply participants.tsv:261]: the "newline" character at the end of the file has been added somehow automatically, I will restore this line to original one.

Comment (jcohenadad) [reply participants.tsv:261]: what software are you using?

Comment (valosekj) [reply participants.tsv:261]: So far, I edited the `participants.tsv` file directly on GitHub

**Comment by jcohenadad** [comment]
@valosekj can you please open this PR on https://github.com/spine-generic/data-multi-subject

**Comment by valosekj** [comment]
Closed and replaced by https://github.com/spine-generic/data-multi-subject/pull/16

**Pull request closed without merging**